### DISCUSSION TITLE
korean input spacing bug

### DISCUSSION THREAD

**Discussion by pnutbtr1** [ppy/osu-framework#5045] [Q&A] [answered]
When entering Korean, spacing is placed before last character.
I'm using Arch Linux with fcitx and X11.

https://user-images.githubusercontent.com/94981525/154814822-0dc5dac4-b653-47ef-9c55-0131f01a3cb3.mp4

[logs.tar.gz](https://github.com/ppy/osu/files/8102818/logs.tar.gz)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu-framework/issues/5046
